apps/test/unit/templates/SchoolDataInputsTest.jsx

```diff
@@ -1,132 +1,175 @@
-import {render, screen, fireEvent} from '@testing-library/react';
+import '@testing-library/jest-dom';
+import {fireEvent, render, screen} from '@testing-library/react';
 import React from 'react';
 
+import {CLICK_TO_ADD, NO_SCHOOL_SETTING} from '@cdo/apps/schoolInfo/constants';
 import SchoolDataInputs from '@cdo/apps/templates/SchoolDataInputs';
-import {NO_SCHOOL_SETTING} from '@cdo/apps/templates/SchoolZipSearch';
 import i18n from '@cdo/locale';
 
 describe('SchoolDataInputs', () => {
-  function renderDefault(propOverrides = {}) {
-    render(<SchoolDataInputs {...propOverrides} />);
-  }
+  const mockSetSchoolId = jest.fn();
+  const mockSetCountry = jest.fn();
+  const mockSetSchoolName = jest.fn();
+  const mockSetSchoolZip = jest.fn();
 
-  it('displays headers in basic component render', () => {
-    renderDefault();
-    expect(screen.queryByText(i18n.censusHeading()));
+  beforeEach(() => {
+    jest.clearAllMocks();
   });
 
-  it('does not display headers if includeHeaders prop is false', () => {
-    renderDefault({includeHeaders: false});
-    expect(screen.queryByText(i18n.censusHeading())).toBeFalsy();
+  it('should render with initial props and default state', () => {
+    render(
+      <SchoolDataInputs
+        schoolId="123"
+        country="US"
+        schoolName="School Name"
+        schoolZip="12345"
+        schoolsList={[{value: '1', text: 'School 1'}]}
+        schoolZipIsValid={true}
+        setSchoolId={mockSetSchoolId}
+        setCountry={mockSetCountry}
+        setSchoolName={mockSetSchoolName}
+        setSchoolZip={mockSetSchoolZip}
+      />
+    );
```

Comment:
Nit: could this be pulled out into a `renderDefault` function that accepts `overrideProps` so it's clearer when scanning through tests which props are being changed to update the given output?

Reply:
ah, nice call! will update to use that function

Reply:
updated!